e2e/integration/apicatalogtests.js

```diff
@@ -6,6 +6,7 @@ Cypress.on( 'uncaught:exception', ( err, runnable ) => {
 } )
 context( 'API Catalog Tests', () => {
     before( () => {
+        cy.viewport( 1280, 720 );
```

Comment:
@mudit94 if all the viewports have the same value maybe using a variable would make sense right?

Reply:
Hi @hybridx , viewports are basically browser window size which I configured to make it maximize

Reply:
@riginoommen can you please double check with @mudit94 about using a single variable across different files

Reply:
hi @hybridx @riginoommen fixed in https://github.com/1-Platform/one-platform/pull/1355/commits/1a3e42f2a6e43249661271f159a25098d88d884b